code/modules/projectiles/projectile/special.dm

```diff
@@ -354,4 +354,36 @@ obj/item/projectile/kinetic/New()
 		return ..()
 	if(!gun)
 		qdel(src)
-	gun.create_portal(src)
\ No newline at end of file
+	gun.create_portal(src)
+
+/obj/item/projectile/snowball
+	name = "snowball"
+	icon_state = "snowball"
+	hitsound = 'sound/items/dodgeball.ogg'
+	damage = 3
+	damage_type = BURN
+
+/obj/item/projectile/snowball/on_hit(atom/target)	//chilling
+	if(istype(target, /mob/living))
+		var/mob/living/M = target
+		M.bodytemperature = max(0, M.bodytemperature - 50)	//each hit will drop your body temp, so don't get surrounded!

```

Comment:
The mission is meant to be lethal to under-prepared / solo explorers. The snowballs are able to be dodged, and there are a fair number of scenery pieces to use as shields.

In my testing, the snowballs were far less of a threat than the stationary turrets, and the ranged snowmen tended to be spaced enough where you could easily avoid dealing with two or more throwers at a time. Normal snowmen only use melee attacks.

Additionally, this mission is meant for a team of players, so aggro will likely be split anyways. There are also a number of space heaters on the map that can help players raise their body temps should they take too many snowballs to the face.